src/conversion/temperature/fahrenheit.js

```diff
@@ -0,0 +1,12 @@
+const shared = require('../shared');
+const temperature = require('./_temperature');
+
+module.exports = {
+  "imperialUnits" : [/(?:°|degrees?) ?(?:f|fahrenheit)/, /fahrenheit/],
+  "weakImperialUnits" : ["f", "degrees?"],
+  "standardInputUnit" : "°F",
+  "isInvalidInput" : (i) => false,
+  "isWeaklyInvalidInput" : (i) => i > 1000,
```

Comment:
just noticed this is missing an `isHyperbole` test

Reply:
Ah, I think it wasn't there because 10000 is over 1000 anyways so it will already return true.

Reply:
Yeah potentially we could have more conditions in `isHyperbole`, but it's fine for now I guess.